x-pack/metricbeat/tests/system/test_appsearch.py

```diff
@@ -0,0 +1,33 @@
+from xpack_metricbeat import XPackTest
+import metricbeat
+import unittest
+
+
+class Test(XPackTest):
+    COMPOSE_SERVICES = ['appsearch']
+    COMPOSE_TIMEOUT = 600
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_status(self):
```

Comment:
Probably should rename this to `test_stats` since that's the metricset being tested.